phpBB/includes/acp/acp_update.php

```diff
@@ -39,6 +39,9 @@ function main($id, $mode)
 		{
 			$recheck = $request->variable('versioncheck_force', false);
 			$do_update = $request->variable('do_update', false);
+			$token = $request->variable('form_token', false);
```

Comment:
Since token is a string shouldn't we kind of default to ''?

Reply:
it is easier to do the check if the default is false, does this break any parsing/validation?